### ISSUE TITLE
[css-images] Integrate with SVG Integration spec

### ISSUE THREAD

**Issue by upsuper** [w3c/csswg-drafts#4640]
The [SVG Integration](https://svgwg.org/specs/integration/) defines several referencing mode for how SVG should be used.

For the concern of the CSS Images spec, SVG should probably be using the animated image document referencing mode, which means external resources and scripts are disallowed.

I believe this is already the practice in all browsers, but it's not mentioned in the spec.

(HTML has a similar issue in whatwg/html#702)

**Comment by svgeesus** [comment]
> For the concern of the CSS Images spec, SVG should probably be using the animated image document referencing mode, which means external resources and scripts are disallowed.

Yes. Note that this also disallows external stylesheets.

P.S. I believe the [svg integration processing modes](https://svgwg.org/specs/integration/#processing-modes) were folded into the SVG2 spec as [processing modes](https://svgwg.org/svg2-draft/conform.html#processing-modes).

**Comment by upsuper** [comment]
Yes, the processing modes have been folded into SVG2 spec (but they have some differences), but the reference modes have not.

**Comment by svgeesus** [comment]
Hmm, that seems suboptimal. There should be one source of truth for a thing. If the reference modes remain in a separate spec, the processing modes should be removed and reference SVG2.
